src/webui/www/private/scripts/client.js

```diff
@@ -462,6 +487,53 @@ window.addEvent('load', function() {
             children[i].className = (children[i].id === selectedTag) ? "selectedFilter" : "";
     };
 
+    const updateTrackerList = function() {
+        const trackerFilterList = $('trackerFilterList');
+        if (trackerFilterList === null)
+            return;
+
+        while (trackerFilterList.firstChild !== null)
+            trackerFilterList.removeChild(trackerFilterList.firstChild);
+
+        const createLink = function(hash, text, count) {
+            const html = '<a href="#" onclick="setTrackerFilter(' + hash + ');return false;">'
```

Comment:
is `return false;` required?

Reply:
It prevents the page from navigating to a blank url fragment (`#`).